# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/sync15/src/client/mod.rs
# Commit: 57b2cda7e0db
# Full Hash: 57b2cda7e0db8b07fcc1e09113ce26add178c68a
# Author: Sammy Khamis <skhamis@mozilla.com>
# Date: 2023-03-01 03:56:13
# Regressor Bug: 1818349
# File Overlap Count: 23
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Bug 1818349 Part 1: Vendor new version of application-services r=markh
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D171181
# ==============================================================================

diff -r eaf9f12a1481 -r 57b2cda7e0db third_party/rust/sync15/src/client/mod.rs
--- a/third_party/rust/sync15/src/client/mod.rs	Tue Feb 28 04:22:35 2023 +0000
+++ b/third_party/rust/sync15/src/client/mod.rs	Tue Feb 28 05:14:29 2023 +0000
@@ -25,7 +25,7 @@
 mod token;
 mod util;
 
-pub(crate) use coll_state::CollState;
+pub(crate) use coll_state::{CollState, LocalCollStateMachine};
 pub(crate) use coll_update::{fetch_incoming, CollectionUpdate};
 pub(crate) use collection_keys::CollectionKeys;
 pub(crate) use request::InfoConfiguration;